### PR TITLE
Feat/summary tile

### PR DESCRIPTION
**Pull request by elliotjordan1**
Creates the summary tile.

Includes mockData.json and assets. Testing at 100%. Excludes functionality that will come with the modal. 

Also adds --coverage for jest runs and other configurations

### CODE REVIEW

File: src/assets/mockData.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+  "projects":[
+    {
+      "id": "0",
+      "name": "Best Project",
+      "client": "Big Boss",
+      "imageURL": "https://www.bfi.org.uk/sites/bfi.org.uk/files/styles/full/public/image/luminous-2013-024-2001-poster.jpg?itok=CQlMhX-H",
+      "blurb": "project described here",
+      "Tech Stack": ["React", "Nodejs"],
```
Comment (jaiminchouhan) [inline src/assets/mockData.json]: key should be techStack

File: src/assets/mockData.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
+{
+  "projects":[
+    {
+      "id": "0",
```
Comment (jaiminchouhan) [inline src/assets/mockData.json]: should this be a project id?

File: src/assets/mockData.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      "name": "Best Project",
```
Comment (jaiminchouhan) [inline src/assets/mockData.json]: projectName

Comment (jaiminchouhan) [reply src/assets/mockData.json]: actually name may be okay

File: src/assets/mockData.json (hunk 2 above)
Comment (elliotjordan1) [reply src/assets/mockData.json]: yes

File: src/assets/mockData.json (hunk 3 above)
Comment (elliotjordan1) [reply src/assets/mockData.json]: left as name, to keep consistency with client, blurb etc (think it's ok as project.name etc when accessing)

**Pull request merged**